### PR TITLE
Added custom actions.

### PR DESCRIPTION
**Pull request by stepek**
ESI-10662

### CODE REVIEW

**Comment by thayashi** [comment]
I'd like to change the layout like this:
screen shot 2017-02-02 at 8 41 30 pm

**Comment by thayashi** [comment]
Also, I'd like to change the dialog like
dialog

**Comment by stepek** [comment]
screenshot-localhost 8080-2017-02-03-08-50-03
screenshot-localhost 8080-2017-02-03-08-50-31
Now should be ok

File: app/templates/detail.html [hunk 1]
```diff
@@ -4,9 +4,19 @@
     <div class="row gohan-detail-header">
       <div class="col-sm-8 resource-name"><h2><%- schema.title %></h2></div>
       <div class="col-sm-4 actions">
-        <a class="edit" data-gohan="update" href="#"><span class="glyphicon glyphicon-pencil" aria-hidden="true"></span>Edit</a><a class="edit" data-gohan="delete" href="#"><span class="glyphicon glyphicon-trash" aria-hidden="true"></span>Delete</a>
+        <a class="edit" data-gohan="update" href="#"><span class="glyphicon glyphicon-pencil" aria-hidden="true"></span>Edit</a>
+        <a class="edit" data-gohan="delete" href="#"><span class="glyphicon glyphicon-trash" aria-hidden="true"></span>Delete</a>
+<%
+      if (hasActions) {
+%>
+        <span data-gohan="actions"></span>
```
Comment (thayashi) [inline app/templates/detail.html]: <span data-gohan="actions" class="edit"></span>

Comment (thayashi) [inline app/templates/detail.html]: <span data-gohan="actions" class="edit"></span>

Comment (thayashi) [inline app/templates/detail.html]: test

Comment (thayashi) [inline app/templates/detail.html]: <span data-gohan="actions" class="edit"></span>

**Comment by thayashi** [comment]
LGTM

**Pull request merged**